inc/saiswitch.h

```diff
@@ -2787,6 +2787,48 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_ARS_PROFILE,
 
+    /**
+     * @brief Available IP Next hop group entries
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_IP_NEXT_HOP_GROUP_ENTRY,
+
+    /**
+     * @brief Available IP Next hop group member entries
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_IP_NEXT_HOP_GROUP_MEMBER_ENTRY,
+
+    /**
+     * @brief Available number of VOQ in the VOQ System (chassis)
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_SYSTEM_VOQS,
+
+    /**
+     * @brief Maximum number of system ports in the switch
+     * Configured system ports can be queried using SAI_SWITCH_ATTR_NUMBER_OF_SYSTEM_PORTS
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_MAX_SYSTEM_PORTS,
+
+    /**
+     * @brief Maximum number of fabric ports in the switch
+     * Configured fabric ports can be queried using SAI_SWITCH_ATTR_NUMBER_OF_FABRIC_PORTS
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_MAX_FABRIC_PORTS,
```

Comment:
can we add validonly SAI_SWITCH_ATTR_TYPE == SAI_SWITCH_TYPE_VOQ`

Reply:
read only attributes can not have condition or validonly check